Arrays/two_sum.js

```diff
@@ -0,0 +1,19 @@
+//Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to target.
```

Comment:
This is brute force solution, could you also add optimal solution using HashTable?